doc/docusaurus/docs/auction-smart-contract/end-to-end/mint.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 15
+---
+
+# Minting the Token to be Auctioned
+
+Before we can start the auction, we need to mint a token to be auctioned.
+To do so, we first must determine the token's currency symbol and name.
+To mint or burn tokens with a specific currency symbol (`currencySymbol`), a Plutus script whose hash matches `currencySymbol` must be provided, and is used to validate the minting or burning action.
+Therefore, we'll first write the on-chain script, then compute its hash and use it as the currency symbol.
+
+## On-chain Minting Policy Script
+
+The full minting policy code can be found at [AuctionMintingPolicy.hs](https://github.com/IntersectMBO/plutus-tx-template/blob/main/src/AuctionMintingPolicy.hs).
```

Comment:
I noticed that minting policy uses Plutus V2. Do we have a task to update it to the Plutus V3?

Reply:
I think V2 is fine for this example, since there's no advantage (in fact there's disadvantage) to use Plutus V3. But if anyone wants to update the example to V3, you are welcome to.